### DISCUSSION TITLE
蛋白也能淡化海水

### DISCUSSION THREAD

**Discussion by delia07152** [1101-GS4719-C/Python-Community-GS4719-C#842] [General]
在某些缺水國家，淡化海水是很重要的一件事，但淡化處理的成本卻非常高，因此水資源非常珍貴。而現在發現蛋白的利用價值後，不論是水資源問題、能源問題以及處理成本問題等等都能迎刃而解，不過剩下的蛋黃該怎麼處理感覺也是個問題。

https://tw.news.yahoo.com/%E7%A0%94%E7%A9%B6%E7%99%BC%E7%8F%BE-%E8%9B%8B%E7%99%BD%E4%B9%9F%E8%83%BD%E6%B7%A1%E5%8C%96%E6%B5%B7%E6%B0%B4-%E6%95%88%E7%8E%87%E6%9B%B4%E5%8B%9D%E6%B4%BB%E6%80%A7%E7%A2%B3%E9%AB%98%E9%81%94-99-020253579.html

**Comment by Zoe06060606** [comment]
這樣水資源匱乏的問題應該能解決不少

**Comment by Jasper525** [comment]
可以跟地方餐廳合作，很多料理都只用蛋黃，而產生蛋白的浪費

**Comment by bonny123865** [comment]
好酷的新技術，蛋黃可以拿去做蛋糕!!!!!
如果有剩下的話，我吃 (沒 我吃不完那麼多

**Comment by Kotori0501** [comment]
如果能運用此技術，我個人覺得好酷喔！
現在台灣有些地區有缺水的現象，如果能好好用運用蛋白進行海水淡化，就有效解決人民用水
至於蛋黃的相關處理...我就不多說了

**Comment by manipulateed** [comment]
蛋黃處理感覺不會問題太大畢竟營養價值很高用途也很廣，至於此項技術居然是用蛋白完成 看來生活中平凡的東西也有可能蘊藏很大的威力呢

**Comment by ching020721** [comment]
若有發生蛋過量的情況的話，剩餘的蛋白拿去做海水淡化還不錯，而蛋黃也如樓上所說可以拿來做烹飪，不過在前陣子蛋供不應求的情況下，民生需求可能還是需要先解決的

**Comment by williamyen0623** [comment]
蛋黃肯定很多商家或家戶需要

**Comment by peggy0329** [comment]
這個新研究太酷了！我覺得除了可以讓雞蛋發揮其他的用途
若只需要用到蛋黃的店家(如：蛋糕店)，就可以解決剩食問題
或許政府可以考慮透過雞蛋供過於求時買入雞蛋淡化海水，也可以解決雞蛋價格大漲的問題
淡化的海水可以進行儲存，當春天缺水時就可以用這些淡化水了
是一個可以解決水資源與物價問題的新方法！

**Comment by evanchou1204** [comment]
確實是低成本，但這樣蛋不會很容易臭掉嗎？

**Comment by CPCpyt** [comment]
這是我從未聽過的構想，真的好令我驚訝原來蛋白也能淡化海水。
不過在實務上要怎麼運用還是很好奇的。

**Comment by StevenWanglolz** [comment]
誰能想到蛋白竟然也有這種功效，太神奇了吧!

**Comment by Tuoresa** [comment]
太酷了吧！！
發現的研究員也太神奇哈哈
好好奇真正應用到海水淡化上會是如何

不知道能不能發展出人工製造純蛋白相似物質的科技
解決了蛋黃的問題，也能提升效率的感覺

**Comment by swerfg03joe** [comment]
利用蛋白來淡化海水，還真是前所未聞，不過使用蛋白成本會不會太高，就算他比活性碳的效率還高上不少，用食物作替代可能不是太合適，剩餘的蛋黃處理也是一大問題
